solidus_quiet_logistics.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
   s.description = s.summary
   s.license     = 'BSD-3-Clause'
 
-  s.required_ruby_version = '~> 2.5'
+  s.required_ruby_version = '~> 2.6'
```

Comment:
Why this change?

Reply:
This was an oversight, we were discussing the bump and I intended to do it later in a different PR.

Reply:
This does not belong in this PR and will be removed